src/Oscoin/Telemetry/Logging.hs

```diff
@@ -454,8 +454,7 @@ fprocessId = F.mapf (fromIntegral @CPid @Int32) (ftag "pid" % F.int)
 -- | When given a function from 'a' to an error message 'Text', format
 -- a domain-specific error as @error_class=foo_bar error_message=\"The error\"@
 -- where the @error_class@ is generically derived from the data constructors.
-ferror :: (SOP.Generic a, SOP.HasDatatypeInfo a)
-       => (a -> Text) -> Format r (a -> r)
+ferror :: SOP.HasDatatypeInfo a => (a -> Text) -> Format r (a -> r)
```

Comment:
Interesting, so looks like newer versions of `generics-sop` do not necessitate anymore on the `SOP.Generic` constraint? Good to know 🤔